packages/react-components/src/templates/__tests__/SharedResearchProposal.test.tsx

```diff
@@ -68,11 +68,11 @@ it('falls back to created date when added date omitted', () => {
 
 it('displays contact pm card when there are contact emails', () => {
```

Comment:
Test name

Reply:
Leaving this as is, as we are asserting on the word "contact pm"